apps/web/utils/actions/invite-member.validation.ts

```diff
@@ -0,0 +1,8 @@
+import { z } from "zod";
+
+export const inviteMemberBody = z.object({
+  email: z.string().email("Please enter a valid email address"),
+  // skipping owner here as the first user that created the org is already the owner
```

Comment:
so this means we can't have 2 owners is the idea? probs make sense to have multiple owners, esp for large orgs.